camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -364,7 +364,11 @@ def on_model_change(self, model_select_val):
 
     def on_scale(self, scalevalue):
         if self.c.calibrated:
-            self.c.set_alpha(scalevalue / 100.0)
+            if self.c.camera_model is not CAMERA_MODEL.FISHEYE:
```

Comment:
Nit: you could merge the two ifs together. I believe it's better for readability.

Reply:
changed